flake.nix

```diff
@@ -25,7 +25,7 @@ rec {
       channels.nixpkgs.overlaysBuilder = channels: [
         rust-overlay.overlays.default
         (final: prev: {
-          inherit (channels.nixpkgs-unstable) wasm-bindgen-cli;
+          #          inherit (channels.nixpkgs-unstable) wasm-bindgen-cli;
         })
```

Comment:
I there a reason for not removing this altogether?

Reply:
So that we can more easily put it back in, once the version in stable nixpkgs gets inevitably out of date. But if you prefer, yeah, I can also remove this (and the whole `nixpkgs-unstable` input with it).

Reply:
I've moved this into its own commit ([ee0b9d2](https://github.com/knowsys/nemo/pull/561/commits/ee0b9d286296165769f2981230a29fe4a9bf2711)), which can be reverted should the need arise.

Reply:
Thanks!
I would also be fine with leaving it but putting explanation why this is commented out :)